linkedevents/settings.py

```diff
@@ -658,7 +659,8 @@ def dummy_haystack_connection_for_lang(language_code):
 
 # A list of hex-encoded 32 byte keys used for encrypting sensitive data
 FIELD_ENCRYPTION_KEYS = env("FIELD_ENCRYPTION_KEYS")
-
+# Specify the number of days after which signup and signup groups will be anonymized
```

Comment:
A minor thing: adding an empty line before and after the new setting and its comment would improve the readability IMO

Reply:
Added empty lines before and after to improve readability